sdks/java/io/mongodb/src/main/java/org/apache/beam/sdk/io/mongodb/MongoDbIO.java

```diff
@@ -312,7 +318,10 @@ public Read withBucketAuto(boolean bucketAuto) {
       return builder().setBucketAuto(bucketAuto).build();
     }
 
-    /** Sets a queryFn. */
+    /**
+     * Sets a queryFn. The provided queryFn must be one of the predefined classes in the MongoDbIO
+     * package such as FindQuery or AggregationQuery.
```

Comment:
You should be able to link to the actual objects like https://github.com/apache/beam/blob/f4204fbeb9daeb6c9261c3b424ed84870eb702ed/sdks/java/io/mongodb/src/main/java/org/apache/beam/sdk/io/mongodb/MongoDbIO.java#L892

That will create links like https://beam.apache.org/releases/javadoc/2.2.0/org/apache/beam/sdk/io/mongodb/MongoDbIO.Read.html#:~:text=%3Foptions%20are%20connection,withMaxConnectionIdleTime(int).

Reply:
done, thanks